.github/workflows/pr_shelter.yml

```diff
@@ -114,7 +115,7 @@ jobs:
       shell: bash
 
     - name: Run enclave tls server for null quote
-      run: docker exec $inclavare_test bash -c 'cd /opt/enclave-tls/bin/ && ./enclave-tls-server -a nullquote -i 127.0.0.1 -p ${{ steps.random-port-generator1.outputs.random-port }}' &
+      run: docker exec $inclavare_test bash -c 'cd /opt/enclave-tls/samples && ./enclave-tls-server -a nullquote -i 127.0.0.1 -p ${{ steps.random-port-generator1.outputs.random-port }}' &
```

Comment:
It should be `/usr/share/enclave-tls/samples` rather than `/opt/enclave-tls/samples`